app/models/comment.rb

```diff
@@ -143,6 +143,13 @@ def title(length = 80)
     HTMLEntities.new.decode(truncated_text)
   end
 
+  # A helper method to encode logic formerly part of spam detection.
+  def from_recently_registered_user?
+    return true unless user&.registered_at
+
+    user.registered_at <= 5.days.ago
```

Comment:
We do have `Settings::RateLimit.user_considered_new_days`, should we maybe reuse this instead of hardcoding a different value here? This would make things a bit more consistent across the app. We also have this nicely wrapped in `UserDecorator#considered_new?`:

```ruby
  def considered_new?
    min_days = Settings::RateLimit.user_considered_new_days
    return false unless min_days.positive?

    created_at.after?(min_days.days.ago)
  end
```

I'm usually against adding more things to `User` but if you really end up reusing this it probably would make sense to move it from the decorator to here.